src/sdt.rs

```diff
@@ -117,6 +118,14 @@ where
          * and length, and then the dispatched to the correct function to actually parse the table.
          */
         match signature {
+            "FADT" => {
+                let fadt_mapping = handler.map_physical_region::<Fadt>(physical_address);
+                match parse_fadt(&fadt_mapping) {
+                    Ok(()) => {}
+                    Err(err) => panic!("Error parsing FADT: {:#?}", err),
+                }
+                handler.unmap_physical_region(fadt_mapping);
+            }
```

Comment:
I think you'll need a comma here after the match arm

Reply:
Not necesseraly, but required when using one line statements. And rustfmt removes it so .. :smile:

Reply:
Huh, fair enough